runtime/onert/backend/cpu/ops/PadLayer.h

```diff
@@ -41,17 +41,16 @@ class PadLayer : public ::onert::exec::IFunction
 public:
   template <typename T> void padImpl(const T *constant_value_data);
 
-  void configure(const IPortableTensor *input, IPortableTensor *output, const int32_t *padData,
-                 int32_t padRank, const void *constantValueData = nullptr);
+  void configure(const IPortableTensor *input, const IPortableTensor *pad,
+                 const IPortableTensor *value, IPortableTensor *output);
 
   void run() override;
 
 protected:
   const IPortableTensor *_input;
+  const IPortableTensor *_pad;
+  const IPortableTensor *_value;
```

Comment:
(Q) Does `_value` mean `Pad data`?

Reply:
No, it's a pad value in scalar according to https://www.tensorflow.org/api_docs/python/tf/pad. Default pad value is 0. If pad value is 1, the kernel will fill pad with 1.

Reply:
@ragmani 

Then, does this `_value` mean a combination of `mode` and `constant_values` in TensorFlow guideline?

Reply:
Circle and tflite don't seem to support `mode`. Pad op and PadV2 op don't have any attributes(option). Then, `_value` is always constant value.

Reply:
AFAIK, (I'm not certain this is @jyoungyun 's point)
- `_pad` would have like `[1,1,1,1,1,1,1,1]` which means how many to add padding
- `_value` would have like `[7.f]` which fills the padded elements

So I think `_value` means a constant value (not a combination).

Reply:
According to my understanding, the `_value` is a constant value of which data to fill with padding elements. And `_pad` has a shape and includes a padding value to be added to the original tensor. Am I right? Thank you for you guy's explanation. :)